docs/model-archiver.md

```diff
@@ -135,7 +135,7 @@ Handler can be TorchServe's inbuilt handler name or path to a py to handle custo
 3. `text_classifier`
 4. `image_segmenter`
 
-For more details refer [default handler documentation](../docs/default_handlers.md)
+For more details refer [default handler documentation](https://pytorch.org/serve/default_handlers.html)
```

Comment:
@jlin27 Same comment here to keep all the links relative and not hardcode the path

Reply:
This was kept as a hardcoded link because we found that in md files for sphinx, relative links to other document did not resolve correctly.